Experimental/DistributedAuthoritySample/ProjectSettings/TimeManager.asset

```diff
@@ -3,7 +3,11 @@
 --- !u!5 &1
 TimeManager:
   m_ObjectHideFlags: 0
-  Fixed Timestep: 0.02
+  Fixed Timestep:
```

Comment:
This was to test out animations, the timestep can be lowered to see movements in play mode in slow motion. It should be set back to normal now, but I will check and revert the change to develops branch state.

Reply:
Hm I reset them to normal, but it still shows it as a change. @davehunt-unity do these settings look normal to you? Is there something else I should do instead of resetting stats in the "Time" window?